app/workers/deletes.php

```diff
@@ -23,6 +23,10 @@ class DeletesV1 extends Worker
 {
     protected Database $consoleDB;
 
+    public function getWorkerName(): string {
```

Comment:
```suggestion
    public function getName(): string {
```
The word `worker` is already obvious from the context. Not need to mention it again.

Reply:
Renamed all `getWorkerName` to `getName` - in abstract class, in every worker and in log where I used this method.